### ISSUE TITLE
The Cloud Code YAML Support server crashes

### ISSUE THREAD

**Issue by jacek-jablonski** [GoogleCloudPlatform/cloud-code-vscode#553]
Issue Type: <b>Bug</b>

*Context*
- Managed Dependencies: off
- Cloud SDK Version: 377.0.0
- Skaffold Version: Not installed
- Minikube Version: 1.25.2

Hi,
when opening YAML files, I get an error about Cloud Code Yaml Support server crashes:

```

TypeError: Cannot read properties of null (reading 'value')
    at t.YamlValidationsProvider.<anonymous> (/Users/jacek/.vscode/extensions/googlecloudtools.cloudcode-1.17.1/dist/server/server.js:1605:1075)
    at Generator.next (<anonymous>)
    at /Users/jacek/.vscode/extensions/googlecloudtools.cloudcode-1.17.1/dist/server/server.js:1605:312
    at new Promise (<anonymous>)
    at r (/Users/jacek/.vscode/extensions/googlecloudtools.cloudcode-1.17.1/dist/server/server.js:1605:57)
    at t.YamlValidationsProvider.validateCustomSchemaExists (/Users/jacek/.vscode/extensions/googlecloudtools.cloudcode-1.17.1/dist/server/server.js:1605:835)
    at t.YamlValidationsProvider.<anonymous> (/Users/jacek/.vscode/extensions/googlecloudtools.cloudcode-1.17.1/dist/server/server.js:1605:730)
    at Generator.next (<anonymous>)
    at /Users/jacek/.vscode/extensions/googlecloudtools.cloudcode-1.17.1/dist/server/server.js:1605:312
    at new Promise (<anonymous>)
[Error - 12:25:39] Connection to server got closed. Server will not be restarted.
```

Extension version: 1.17.1
VS Code version: Code 1.66.0 (e18005f0f1b33c29e81d732535d8c0e47cafb0b5, 2022-03-30T05:50:04.419Z)
OS version: Darwin x64 21.4.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz (16 x 2300)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 3|
|Memory (System)|32.00GB (7.49GB free)|
|Process Argv|. --crash-reporter-id 8a1774d7-ab93-4aca-b311-a151b6f966f1|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by glouischandra** [comment]
Hello @jacek-jablonski, this looks like some transient error, is there any specific content/actions in the YAML content and is it consistently failing for you?

**Comment by jacek-jablonski** [comment]
Hi @glouischandra,
Nothing specific, just opening some yaml files. I am attaching a screencast, hope it helps

https://user-images.githubusercontent.com/35669512/162388327-574262b2-31e8-4e89-8af8-bfde6ec3d706.mov

.

**Comment by glouischandra** [comment]
Thanks @jacek-jablonski we're still investigating the root cause of this, will get back to you soon.

**Comment by kschaab** [comment]
Fix for this should be live today.

**Comment by jacek-jablonski** [comment]
Seems like the problem is gone for now. Thank you.

**Comment by kschaab** [comment]
This should be resolved as of 1.20.1, please do reopen if you see a regression and thank you for reporting!
